lib/components/ms-toggle/MsBooleanToogle.vue

```diff
@@ -0,0 +1,75 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <div class="ms-boolean-toggle">
+    <ion-text
+      type="button"
+      class="button-view button-medium"
+      :class="{ 'button-disabled': modelValue === Answer.No }"
+      :disabled="modelValue === Answer.Yes"
+      @click="$emit('update:modelValue', Answer.No)"
+    >
+      {{ leftValue }}
+    </ion-text>
+    <ion-text
+      type="button"
+      class="button-view button-medium"
+      :disabled="modelValue === Answer.No"
+      :class="{ 'button-disabled': modelValue === Answer.Yes }"
+      @click="$emit('update:modelValue', Answer.Yes)"
+    >
+      {{ rightValue }}
+    </ion-text>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { Answer } from '@lib/components/ms-toggle/types';
+
+defineProps<{
+  modelValue: Answer;
+  leftValue: string;
```

Comment:
Could make them optional and defaulting to `Yes`/`No`.  MsModal already defines them, make them generic https://github.com/Scille/megashark-lib/blob/cae49fd4774b08b8064a35502133f90cb3484f33/lib/locales/en-US.json#L24